.github/workflows/linux.yml

```diff
@@ -97,6 +97,13 @@ jobs:
           PIP_VER=$(python3 -c "import pip; print(pip.__version__)")
           echo "Using pip version: ${PIP_VER}"
           echo "PIP_CACHE_DIR=${PIP_CACHE_PATH}/${PIP_VER}" >> $GITHUB_ENV
+      - name: Get pip cache info
+        run: |
+          echo "Cache size: "
+          du -h -d2 ${PIP_CACHE_DIR}
+          echo "Cache info: "
+          python -m pip cache info
+        continue-on-error: true
```

Comment:
we need to combine all these steps into custom composite github action (CVS-122781)

Reply:
Yes, I saw it, Do you suggest doing that in this PR?
And getting cache info should be set by special argument as it may takes additional time (1 min on slow machines)

Reply:
and I thought to make this step mandatory to check that cache is enabled